### ISSUE TITLE
Airpods make moonlight die

### ISSUE THREAD

**Issue by PolCPP** [moonlight-stream/moonlight-qt#560]
OS/Machine (MBA M1) 11.2.3, using latest stable (3.1)

Long story short, when you connect the airpods to the mac while moonlight is running instead of changing the audio interface the app crashes.

I think it also happened on intel macs on 2.x. But i rarely used them with my previous machine.

Will test if it also happens with regular bluetooth headphones and edit this.

```
Process:               Moonlight [26005]
Path:                  /Applications/Moonlight.app/Contents/MacOS/Moonlight
Identifier:            com.moonlight-stream.Moonlight
Version:               3.1.0 (3.1.0)
Code Type:             ARM-64 (Native)
Parent Process:        ??? [1]
Responsible:           Moonlight [26005]
User ID:               502

Date/Time:             2021-04-01 18:31:03.852 +0100
OS Version:            macOS 11.2.3 (20D91)
Report Version:        12
Anonymous UUID:        ABC88EA8-94D3-69EB-DEAE-6A75353DBD53

Sleep/Wake UUID:       1101D43A-3798-4012-847E-34D5B54642CA

Time Awake Since Boot: 130000 seconds
Time Since Wake:       27000 seconds

System Integrity Protection: enabled

Crashed Thread:        19

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Application Specific Information:
abort() called
Moonlight(26005,0x17125b000) malloc: Heap corruption detected, free list is damaged at 0x6000013f8c00
*** Incorrect guard value: 219043332147
```

**Comment by cgutman** [comment]
It's probably fixed by this commit in SDL https://github.com/libsdl-org/SDL/commit/1133ea03492006c766f3d92697558127fa5e122f

So we will pick up the fix in our next SDL update.

**Comment by cgutman** [comment]
This was fixed in v3.2.0
